adafruit_register/i2c_struct.py

```diff
@@ -34,17 +37,17 @@ class Struct:
     """
     def __init__(self, register_address, struct_format):
         self.format = struct_format
-        self.buffer = bytearray(1+ustruct.calcsize(self.format))
+        self.buffer = bytearray(1+struct.calcsize(self.format))
         self.buffer[0] = register_address
 
     def __get__(self, obj, objtype=None):
         with obj.i2c_device:
             obj.i2c_device.write(self.buffer, end=1, stop=False)
             obj.i2c_device.read_into(self.buffer, start=1)
-        return ustruct.unpack_from(self.format, memoryview(self.buffer)[1:])
+        return struct.unpack_from(self.format, memoryview(self.buffer)[1:])
```

Comment:
Instead of memoryview could we use the offset parameter to `unpack_from`?

Reply:
I forgot to respond to this comment, sorry.  In an  ideal world i think the line of code could be written 'return struct.unpack_from(self.format, self.buffer, 1)'  Note that users of the function need to provide non-native format to the class instance, because micropython struct.pack and unpack do memory alignment.  The memoryview() function might be a way to get around the difference.  Also note that the setter function does NOT use memberview() in the struct.pack() call, which is inconsistent with the getter function.  Without memberview() to have to call this function with '<H' or '>H' or you will not get what you want.

I would suggest that putting the "register_address" as the first byte is causing the problems with unpack.  I would add a class variable for register_address and make the buffer start at [0:]  instead of [1:]

Reply:
I finally read the README.rst and now understand that storing the register_address as the first byte of the buffer is a good thing, because it save lots of memory.  We could remove the memoryview() but we would have to make sure that all callers are using non-native formats.